src/meson.build

```diff
@@ -41,6 +41,6 @@ meson.add_install_script(meson_make_symlink,
 # It's only around for legacy reasons, see netplan/cli/utils.py: get_generator_path()
 install_symlink(
     'generate',
-    pointing_to: join_paths('..', '..') + join_paths(get_option('prefix'), libexec_netplan, 'generate'),
+    pointing_to: join_paths('..', '..', '..') + join_paths(get_option('prefix'), libexec_netplan, 'generate'),
```

Comment:
This change fixed the "make install" fwiw for me

Reply:
I'm a bit worried about this change. It makes our autopkgtest GH action fail. We need to be careful with meson installation paths, as those can be different on Fedora/Debian/Ubuntu and might need to go hand-in-hand with packaging changes. Can we somehow avoid this change?

Maybe this helps: https://github.com/canonical/netplan/commit/2cded2f2673ce90f1585c9f6f9bb16ca7d81e7a6 (i.e. rebase on top of our recent `main` branch). See https://github.com/canonical/netplan/pull/327 for context.